tests/common_test.go

```diff
@@ -39,20 +39,13 @@ var (
 	remote   client.Remote
 )
 
-// dummyGetCertificate is a GetCertificate function which reads a static cert
-// from disk and simulates latency.
-func dummyGetCertificate(op *protocol.Operation) ([]byte, error) {
-	if string(op.Payload) == "slow" {
-		time.Sleep(time.Second)
-	}
-	return ioutil.ReadFile(serverCert)
-}
-
 type dummySealer struct{}
 
 func (dummySealer) Seal(op *protocol.Operation) (res []byte, err error) {
 	if op.Opcode != protocol.OpSeal {
 		panic("wrong op")
+	} else if string(op.Payload) == "slow" {
+		time.Sleep(time.Second)
```

Comment:
What's this about?

Reply:
It's for TestConcurrency. We send a slow request, followed by a fast request, and want the response to the fast request to come back first. This functionality used to be in dummyGetCert.